docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md

```diff
@@ -2,21 +2,60 @@
 title: Pod Security Standards
 ---
 
+[Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) define security restrictions for a broad set of workloads.  They replace [Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/), which are now deprecated.
 
-[Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) are a built-in replacement for Pod Security Policies which broadly define security restrictions. 
 
+## Upgrade to Pod Security Standards
+Make sure that all Pod Security Policies were migrated, which should result in equivalent security controls still being enforced (e.g. Pod Security Admission, Gatekeeper, Kyverno) and all PodSecurityPolicy objects removed from the cluster.
 
+:::important
+The new policy enforcement must be added before the PodSecurityPolicy objects are removed, or risk creating a window of opportunity for privilege escalation within the cluster.
 
+## Pod Security Admission Configuration Templates
 
+Rancher comes with two Pod Security Admission Configuration (PSAC) templates that you can assign to a cluster:
+* `rancher-privileged`:  This is the most permissive configuration and doesn't restrict the behavior of any pod.
+* `rancher-restricted`:  This policy follows current pod-hardening best practices.
 
 
+If you are a Rancher administrator or have restricted administrator privileges, you can customize restrictions and permissions by creating additional PSAC templates, or by editing existing templates.
 
+:::caution
+If you edit an existing PSAC template while it is still in use, it *will* affect all clusters that have been assigned that template.
+:::
 
-###  TODO
+If you want to allow users other than the Rancher administrator to manage PSAC templates, you can bind the user to a role that grants the user all verbs ("*") on `management.cattle.io/podsecurityadmissionconfigurationtemplates`.
 
-* PSAC templates
-	* How they work, what the flow is
-	* That updating a template will affect live clusters
-	* Who can edit. Including a template management role workaround with it's own warning/caveat.
-* How to upgrade
-	* Ensure that a replacement is in place before you upgrade and PSPs are off
\ No newline at end of file
+:::warning
+Any user that is bound to the above permission will be able to change the restriction levels on all managed clusters which use this template, including ones they have no permissions on.
+:::
+
+
+### Rancher On PSA-restricted Clusters
+
+When you run Rancher on a Kubernetes cluster that enforces the `rancher-restricted` PSS policy by default, you'll need to exempt the following namespaces. This prevents the policy from blocking Rancher system pods:
+
+Here is the list of the namespaces that require an exemption:
+* `kube-system`
```

Comment:
For consistency with our other content, could you please use `-` to start the items in this list. `-` is preferable to `*` for readability since `*` is used to italicize content.